### ISSUE TITLE
Sheet single selection keyboard a11y issues

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2237]
## Description

The current definition of the sheet component is to be dismissed after a user selects an option in a single selection list.

When using a keyboard, in order to move inside a radio group the options are selected. This forbid users to reach kore than the second option if none is selected.

The expected result is to prevent the dismiss of the sheet until the user has made its desired selection. If not, the user will be unable to reach all radio group options.

## Tasks

- [ ] Update specs
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [ ] Android ticket
- [ ] iOS ticket
- [ ] Web ticket
